UserInterface/ViewModels/MainWindowViewModel.cs

```diff
@@ -153,4 +165,17 @@ private void PreviousViewModelChanged()
     /// This method is only used to make sure the application can't be stopped using alt+f4
     /// </summary>
     private void DoNothing() { }
+
+    private void GetColorSetting()
+    {
+        Settings savedSettings = settingService.GetSetting();
+        System.Windows.Media.Color color = (System.Windows.Media.Color)System.Windows.Media.ColorConverter.ConvertFromString(savedSettings.Color);
+        SolidColorBrush solidColorBrush = new SolidColorBrush(color);
+
+        ResourceDictionary resourceDict = new ResourceDictionary();
+        resourceDict.Source = new Uri("../../Assets/Styling/Colors.xaml", UriKind.RelativeOrAbsolute);
```

Comment:
Is het niet beter om dit in app.xaml.cs te definiëren? Want blijft die App.Current.Resource altijd in memory? Hoe werkt dit precies?

Reply:
Zie mijn comment in andere pull request.

Reply:
ja, laat dit maar zitten voor nu anders